src/components/calcite-dropdown/calcite-dropdown.e2e.ts

```diff
@@ -484,7 +484,7 @@ describe("calcite-dropdown", () => {
 
   it("should focus the first item on open when there is no active item", async () => {
     const page = await newE2EPage({
-      html: `<calcite-dropdown>
+      html: `<calcite-dropdown style="--calcite-popper-transition:none;">
```

Comment:
@jcfranco this is how I'm getting around the animation killing the build. Is this acceptable?

Reply:
Let's proceed with this for now. It could get messy if we have different transition vars across components.